charts/kubearchive/values.yaml

```diff
@@ -51,18 +42,22 @@ sink:
   # a json path that if it returns a value, will cause the sink to delete the resource
   deleteWhen: ".status.completionTime"
 
+# values used to create the operator
 operator:
+  name: "{{ .Release.Name }}-operator"
   # NOTE - Helm does not resolve the `ko` path. The path needs to be set on the `helm install` command.
   # For example:
   # helm install -n default <chart-name> charts/kubearchive \
   # --set-string operator.image=$(ko build github.com/kubearchive/kubearchive/cmd/operator)
   image: ko://github.com/kubearchive/kubearchive/cmd/operator
+  metricsPort: 8443
```

Comment:
Why are these still exposed while all the other vars have gone elsewhere?

Again, I think these are unlikely to ever change, certainly not by a user.

Reply:
Fixed. I also removed the ports